tables/tr-g2.tbl

```diff
@@ -0,0 +1,17 @@
+#+locale:tr
+#+type:literary
+#+contraction:full
+#+grade:2
+#+dots:8
+
+# TODO: Please correct the metadata above. It is not meant to be
```

Comment:
So you're saying that you don't know if the meta data is correct? I'm confused: I would assume that you know the meta data since you know the language and the system you created the table for.

If your meta data is correct then please remove the TODO